en/developer_guide/specific_backbone_view.rst

```diff
@@ -0,0 +1,37 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+In some case, during the integration of the Open Orchestra Backoffice, the generic view used for the
+edition page is not enough. There are some new behaviour to add to the form, such as a specific event
+when you edit a field.
+
+There are two simple solution to avoid :
+
+ - Add some javascript in the form template
+ - Add a global listener to the javascript project
+
+Both of these solution are to avoid because :
+
+ - They force a link between the templating and the javascript
+ - The may be checked each time the dom is modified
+
+To prevent these usage, Open Orchestra provide a way to modify the view used in Backbone.
```

Comment:
usages, provides
i would rephrase the end : the Backbone view used by Orchestra